dev/tests/settings_local.py

```diff
@@ -17,8 +17,8 @@
 }
 
 IDSUBMIT_IDNITS_BINARY = "/usr/local/bin/idnits"
-IDSUBMIT_REPOSITORY_PATH = "test/id/"
-IDSUBMIT_STAGING_PATH = "test/staging/"
+IDSUBMIT_REPOSITORY_PATH = "/assets/ietf-ftp/internet-drafts/"
```

Comment:
This should go into `/assets/ietfdata/doc/draft/archive` to match the tree currently in production at ietfdata (and things that are currently in www6 will move into that tree as we move all of this into a blob-store).

Similarly the repository should be at `/assets/ietfdata/doc/draft/repository`

and other document types are simpler:

```
./doc/bofreq
./doc/charter
./doc/conflict-review
./doc/review
./doc/slides # This one is strange - it's edu + iab ws slides at the moment, not meeting materials slides
./doc/status-change
./doc/statement
```

Reply:
"This should go" - which this? `IDSUBMIT_REPOSITORY_PATH` or `IDSUBMIT_STAGING_PATH`?

Reply:
`IDSUBMIT_REPOSITORY_PATH`

Thanks for asking - my direction above was a little off.
In production right now:
https://github.com/ietf-tools/datatracker/blob/a02812bf7c7177471af18141bcf6cae6375d9ee8/ietf/settings.py#L674
and 
https://github.com/ietf-tools/datatracker/blob/a02812bf7c7177471af18141bcf6cae6375d9ee8/ietf/settings.py#L799

We should mirror that in the dev instance (it's important that the submisssion tool post draft into the repository, not the archive).

That is, we should have this in the dev instance:
`IDSUBMIT_REPOSITORY_PATH = /assets/ietfdata/doc/draft/repository`